src/graphics/aurora/highlightable.h

```diff
@@ -76,6 +78,8 @@ class Highlightable {
 	float _lowerBoundG;
 	float _lowerBoundB;
 	float _lowerBoundA;
+	
```

Comment:
No, the problem is not the empty line, that is good and wanted. But the line should be completely empty. Right now, there's still a (invisible) tab character in that line. Make the line completely empty by removing the tab.